pkg/apis/openebs.io/v1alpha1/cas_template_keys.go

```diff
@@ -14,6 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Package v1alpha1
```

Comment:
This seems meaningless. If we cannot add any info let us remove this.

Reply:
Yes
I had removed it initially. It was a mistake on my side. But the Codecov was giving a negative feedback. So added it back. But the status didn't change. I will remove it.

Reply:
@Tushcr7 
You can remove the comment and put a blank line instead.